tests/unit/amo/components/TestHeroRecommendation.js

```diff
@@ -91,6 +91,32 @@ describe(__filename, () => {
         root.instance().makeCallToActionURL(),
       );
     });
+
+    it('configures an external link to open in a new tab', () => {
+      const _isInternalURL = sinon.stub().returns(false);
+      const shelfData = createShelfData({
+        external: fakePrimaryHeroShelfExternal,
+      });
+
+      const root = render({ _isInternalURL, shelfData });
+
+      const link = root.find('.HeroRecommendation-link');
+      expect(link).toHaveProp('rel', 'noopener noreferrer');
+      expect(link).toHaveProp('target', '_blank');
+    });
+
+    it('does not configure an internal link to open in a new tab', () => {
+      const _isInternalURL = sinon.stub().returns(true);
+      const shelfData = createShelfData({
+        external: fakePrimaryHeroShelfExternal,
```

Comment:
Similar to above, this should set up a realistic internal link with `shelfData.addon` and assert that it gets passed to `_isInternalURL()`. This will help cover the integration of `this.makeCallToActionURL()`.

Reply:
As above, I've made this change. Thanks!